site/content/docs/5.3/content/tables.md

```diff
@@ -80,6 +80,10 @@ Use contextual class to color tables, table rows or individual cells.
 
 ## Accented tables
 
+{{< design-callout-alert info >}}
+It is possible to use striped rows **or** striped columns in your tables, but you **can't use both in a same table**.
```

Comment:
Should we explain why? 'because it does not respect the Orange Design System specifications' for instance?

Reply:
We are not showing examples where we use both in the same table. Is this callout really needed?

Reply:
> Should we explain why? 'because it does not respect the Orange Design System specifications' for instance?

Good point Mewen. Indeed, it would be better to write: "It is possible to use striped rows **or** striped columns in your tables, but you **can't use both in a same table** because  it does not respect the Orange Design System specifications."

Reply:
> We are not showing examples where we use both in the same table. Is this callout really needed?

We thought this design callout message would be necessary to avoid users using both horizontal and vertical striped at the same time, knowing that in the following section "Striped columns", there is an example that could be misleading: it shows 2 striped columns + a striped line (but not the entire line).

Reply:
> there is an example that could be misleading: it shows 2 striped columns + a striped line (but not the entire line).
Yes, this example is really confusing, it is just striped columns, but in the third line, 2 cells are merged with a `colspan="2"`. (example from Bootstrap)

Reply:
IMHO it's obvious that one shouldn't use both at the same time. I vote for not adding this callout. Moreover, the doc doesn't show that they can be used at the same time, it only shows the rendering when there are merged cells.

Reply:
It this `it only shows the rendering when there are merged cells.` that can be confusing to me and motivated my adding the design callout.